app/src/main/java/tech/relaycorp/gateway/background/CogRPC.kt

```diff
@@ -0,0 +1,5 @@
+package tech.relaycorp.gateway.background
+
+object CogRPC {
+    const val PORT = 21473
```

Comment:
Heads up: I'll move this to the core lib: https://github.com/relaycorp/relaynet-jvm/issues/63